libraries/AP_IOMCU/AP_IOMCU.cpp

```diff
@@ -1092,10 +1096,14 @@ void AP_IOMCU::send_rc_protocols()
 /*
   check ROMFS firmware against CRC on IOMCU, and if incorrect then upload new firmware
  */
+#ifndef HAL_IOMCU_FW_FLASH_SIZE
```

Comment:
It would obviously be nicer to always get this from the build / embedding system

Not really needed for this PR.

Reply:
ok, will defer it for another time.